tests/IntlHarness.js

```diff
@@ -10,10 +10,26 @@ import PropTypes from 'prop-types';
 import { IntlProvider } from 'react-intl';
 import messages from '../translations/stripes-smart-components/en_US';
 
+/**
+ * mimics the StripesTranslationPlugin in @folio/stripes-core:
+ * given a list of key-value pairs like {"foo": "bar"} and a prefix,
+ * return {"prefix.foo": "bar", ...}.
+ *
+ * @param {*} obj map of key-value pairs
+ * @param {*} prefix module-path to prepend to each key
+ */
+function prefixKeys(obj, prefix) {
+  const res = {};
+  for (const key of Object.keys(obj)) {
+    res[`${prefix}.${key}`] = obj[key];
+  }
+  return res;
+}
+
 const IntlHarness = ({ children }) => (
   <IntlProvider
     locale="en-US"
-    messages={messages}
+    messages={prefixKeys(messages, 'stripes-smart-components')}
```

Comment:
@zburke I wonder if we could use `https://lodash.com/docs/4.17.15#mapKeys` instead of implementing it by hand but it all looks good.

Reply:
Thanks for the tip; I didn't know about `mapKeys`.